### PR TITLE
Remove load_and_embed_v1 method since it is not used

### LINKED ISSUE #629: refactor: remove `load_and_embed` v1 method from `embedchain` class
### 🐛 Describe the bug

* remove the v1 method which is unused

### PR DESCRIPTION
**Pull request by dev-ayush69**
## Description

Since `load_and_embed_v2` is the method that is being used, `load_and_embed` (v1) has been removed from the `embedchain` class.

Fixes #629 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Refactor (does not change functionality, e.g. code style improvements, linting)
- [ ] Documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

Please delete options that are not relevant.

- [ ] Unit Test
- [ ] Test Script (please provide)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [ ] closes #629 
- [ ] Made sure Checks passed

### CODE REVIEW

File: embedchain/embedchain.py [hunk 1]
```diff
@@ -246,93 +246,7 @@ def add_local(
         )
         return self.add(source=source, data_type=data_type, metadata=metadata, config=config)
 
-    def load_and_embed(
```
Comment (deshraj) [inline embedchain/embedchain.py:249]: Can you please remove the `load_and_embed` function and also rename `load_and_embed_v2` to `load_and_embed` function?

**Review by deshraj — COMMENTED** [review]
Overall looks good. Left a small comment to resolve.

File: embedchain/embedchain.py (hunk 1 above)
Comment (dev-ayush69) [reply embedchain/embedchain.py:249]: Sure

Comment (dev-ayush69) [reply embedchain/embedchain.py:249]: Hey @deshraj , I made the mentioned changes.

Comment (dev-ayush69) [reply embedchain/embedchain.py:249]: @deshraj The mentioned changes are made.

**Review by taranjeet — APPROVED** [review]

**Comment by taranjeet** [comment]
@dev-ayush69 : can you please resolve the conflicts? and let me know when it done.

**Comment by dev-ayush69** [comment]
> @dev-ayush69 : can you please resolve the conflicts? and let me know when it done.

@taranjeet I have resolved the conflicts.

**Comment by taranjeet** [comment]
Thanks.
Can you please make sure that all the tests pass?
@dev-ayush69

**Comment by dev-ayush69** [comment]
@taranjeet Done 👍🏼

**Pull request merged**